File: 240504/수열의 순서 바꾸기/reorder-sequence.py
N = int(input())
n_lst = list(map(int, input().split()))

cnts = 0
for i in range(N):
    flag = False
    for j in range(i, N - 1):
        if n_lst[j] > n_lst[j + 1]:
            flag = True

    if flag:
        cnts += 1

print(cnts)

File: 260501/수열의 순서 바꾸기/reorder-sequence.cpp
#include <bits/stdc++.h>
using namespace std;

int main() 
{
    // Please write your code here.
    int N;
    cin >> N;
    vector<int>n_lst(N);

    for(int i = 0; i < N; i++)
        cin >> n_lst[i];

    int cnts = 0;
    for(int i = 0; i < N; i++)
    {
        bool flag = false;
        for(int j = i; j < N - 1; j++)
        {
            if(n_lst[j] > n_lst[j + 1])
                flag = true;
        }
        if(flag)
            cnts += 1;
    }

    cout << cnts;
    return 0;
}
